src/App.js

```diff
@@ -354,6 +354,12 @@ const codePushOptions = { checkFrequency: codePush.CheckFrequency.MANUAL };
 
 const AppWithCodePush = codePush(codePushOptions)(AppWithSentry);
 
+const PlaygroundWithReduxStore = () => (
+  <ReduxProvider store={store}>
```

Comment:
`Playground` renders `RainbowContextWrapper` which now internally accesses the Redux store, so it was broken until I added this. For clarity I also aliased the `Provider` import as `ReduxProvider` since it's not the only provider in this file.